plugin/src/main/java/net/dzikoysk/funnyguilds/listener/EntityDamage.java

```diff
@@ -29,7 +30,7 @@ public void onDamage(EntityDamageByEntityEvent event) {
             if (config.animalsProtection && (victim instanceof Animals || victim instanceof Villager)) {
                 this.regionManager.findRegionAtLocation(victim.getLocation())
                         .map(Region::getGuild)
-                        .filterNot(guild -> guild.equals(attackerUser.getGuild()))
+                        .filterNot(guild -> guild.equals(attackerUser.getGuild().get()))
```

Comment:
To się może sypnać na .get() jak attacker nie ma gildii, zró attackerUser.getGuild().map(it -> guild.equals(it)).orElse(false)